ProjectHub Android release blogpost (#77)
xmartlabs/blog: _posts/2018-12-20-Introducing-ProjectHub-for-Android.md

---
layout: post
title:  Introducing ProjectHub for Android

> date: 2018-12-20 09:00:00

Reviewer: the date should be the release date

Author: yes

author: Bruno Ferrari

> categories: Android

Reviewer: add more categories (in order to increase the SEO)

author_id: bruno

---

> We have just released our brand-new Android app, **ProjectHub**, to the public. You can download it [here](https://play.google.com/store/apps/details?id=com.xmartlabs.projecthub&hl=en).

Reviewer: something that it's free?

<img width="100%" src="/images/projecthub/projecthub_promo_graphic.png"/>

ProjectHub is an app that lets you manage your GitHub projects. You only need to log in with your GitHub account! In this post, we will talk briefly about the key functionality of the app, so that you can start taking full advantage of it immediately.

#### Browse your repositories

> Browse through all of your repositories, including forks and the ones from organizations. The app also offers filters so that you can select and organization -should you have more than one- to browse only through that organization's repositories.

Reviewer: ```suggestion
 Browse through all of your repositories, including forks and the ones from organizations. The app also offers filters so that you can select an organization -should you have more than one- to browse only through that organization's repositories.
```

#### Import or create projects

> Select an existing project and import it to your device to start working on it or create a new one from scratch. 

Reviewer: add something related to the ability of view a project without internet conection

#### Manage your imported projects

> Create, move, rename and delete columns and cards from your imported projects. Everything you can do from GitHub you can do from ProjectHub.

Reviewer: ```suggestion
Create, move, rename and delete columns and cards from your imported projects. Everything you can do from GitHub you can do easily from ProjectHub.
```

<img width="100%" src="/images/projecthub/projecthub_screenshots.png"/>

> So, what are you waiting for? Download [ProjectHub](https://play.google.com/store/apps/details?id=com.xmartlabs.projecthub&hl=en) and start working on your GitHub projects, anywhere, anytime. If you are an iOS user, don't worry, we got your back! Download ProjectHub for iOS [here](https://itunes.apple.com/uy/app/projecthub/id1245190471?mt=8)!

Reviewer: ```suggestion
 So, what are you waiting for? Download [ProjectHub](https://play.google.com/store/apps/details?id=com.xmartlabs.projecthub&hl=en) and start working on your GitHub projects, anywhere, anytime. If you are an iOS user, don't worry, we got your back! Download ProjectHub for iOS [here](https://itunes.apple.com/uy/app/projecthub/id1245190471?mt=8)!.
```

---
layout: post
title:  Introducing ProjectHub for Android
date: 2018-12-20 09:00:00
author: Bruno Ferrari
categories: Android
author_id: bruno

---

> We have just released our brand-new Android app, **ProjectHub**, to the public. 

Reviewer 2: what about `We have just released to the public our brand-new Android app, **ProjectHub**.` it seems for me a little bit redundant to put to the public at the end.

> We have just released to the public our brand-new Android app, **ProjectHub**. 

Reviewer 3: Shouldn't this statement be like this? We already have ProjectHub, but now we have an Android app for it.

Cause this sounds like PH is new, but the new thing is PH for Android.

Author: true, I'll rewrite it

> You can download it (for free) [here](https://play.google.com/store/apps/details?id=com.xmartlabs.projecthub&hl=en).

Reviewer 3: I personally prefer avoiding the "here"s:

```suggestion
You can [install it from Google Play (for free!)](https://play.google.com/store/apps/details?id=com.xmartlabs.projecthub&hl=en).
```

But it's up to you.

ProjectHub is an app that lets you manage your GitHub projects. 
You only need to log in with your GitHub account! 
In this post, we will talk briefly about the key functionality of the app, so that you can start taking full advantage of it immediately.

#### Browse your repositories

Browse through all of your repositories, including forks and the ones from organizations. 
The app also offers filters so that you can select an organization -should you have more than one- to browse only through that organization's repositories.

#### Import or create projects

Select an existing project and import it to your device to start working on it or create a new one from scratch. 

#### Manage your imported projects

Create, move, rename and delete columns and cards from your imported projects. 
Everything you can do from GitHub you can do easily from ProjectHub.

<img width="100%" src="/images/projecthub/projecthub_screenshots.png"/>

So, what are you waiting for? 
Download [ProjectHub](https://play.google.com/store/apps/details?id=com.xmartlabs.projecthub&hl=en) and start working on your GitHub projects, anywhere, anytime. 
If you are an iOS user, don't worry, we got your back! 

> Download ProjectHub for iOS [here](https://itunes.apple.com/uy/app/projecthub/id1245190471?mt=8)!

Reviewer 3: ```suggestion
Download [ProjectHub for iOS from the App Store](https://itunes.apple.com/uy/app/projecthub/id1245190471?mt=8)!
```
